apps/exit-node/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "rpch-exit-node",
+  "version": "0.0.0",
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "start": "node build/index.js",
+    "build": "tsc",
+    "clean": "rm -rf ./build",
+    "test": "jest || exit 0",
+    "lint": "TIMING=1 eslint \"src/**/*.ts*\""
+  },
+  "devDependencies": {
+    "@types/debug": "^4.1.7",
+    "@types/jest": "^29.2.3",
+    "@types/ws": "^8.5.3",
+    "eslint-config-rpch": "file:configs/eslint/",
```

Comment:
`file:configs/eslint/` may work but is not the right way to do that, you should replace it with `*` similar to how its done in the other packages